<launch>

  <node pkg="lslidar_c32_driver" type="lslidar_c32_driver_node" name="lslidar_c32_driver_node" output="screen">
    <param name="lidar_ip" value="192.168.1.200"/>
    <param name="device_port" value="2368"/>
	<param name="add_multicast" value="false"/>
	<param name="group_ip" value="224.1.1.2"/>
  </node>

  <node pkg="lslidar_c32_decoder" type="lslidar_c32_decoder_node" name="lslidar_c32_decoder_node" output="screen">
    <param name="frame_id" value="laser_link"/>
    <param name="point_num" value="2000"/>
    <param name="channel_num" value="8"/>
    <param name="angle_disable_min" value="0"/>
    <param name="angle_disable_max" value="0"/>
    <param name="angle3_disable_min" value="0"/>
    <param name="angle3_disable_max" value="0"/>
    <param name="min_range" value="0.15"/>
    <param name="max_range" value="150.0"/>
    <param name="frequency" value="10.0"/>
    <param name="publish_point_cloud" value="true"/>
    <param name="publish_scan" value="true"/>
    <param name="use_gps_ts" value="false"/>
  </node>
  

  <!--node name="rviz" pkg="rviz" type="rviz" args="-d $(find lslidar_c32_decoder)/launch/lslidar_c32.rviz" output="screen"/-->

</launch>
